_posts/2017-03-16-two-months-of-tech-interviews.md

```diff
@@ -0,0 +1,220 @@
+
+Then I had some really laid back interviews that did just that - those left me concerned.
+
+> That was all conversational... what if they just hire random people who can talk but can't code?!
+
+> He didn't write any notes. What's he going off of other than just gut instinct?
+
+> This is too easy...
+
+By the end I had a new appreciation for how hard it is to interview.
+
+- You can keep things really conversational, but how does that show if someone can actually code?
+- You can assign a project, but how can you know if the person will have time to do a good job?
+- You can dive into quick algorithm problems, but how much does that actually tell you about the person?
+
+All the while you have a growing stack of applicants just waiting for an interview. 😩
+
+I wonder if any companies let candidates choose. Give a few different options and let the candidate choose the path that works best for them. I'd like to see how that type of system would work out.
+
+### Interview highlights
+
+Luckily, not all my interviews left me banging my head against a wall. Some were really fun! 
+
+- **HackerNews MVP:** One company asked me to build an MVP for HackerNews in 4 hours. I really enjoyed being able to focus on a project without having an interviewer breathing down my neck. The followup was working with some team members to extend the MVP. Since it was my code, I became the most knowledgeable person. Being the one giving directions made the interviewers less intimidating.
+- **Life Story:** Two companies had me tell my life story. Where I grew up, how I got into programming and all the steps that led me to my job hunt. I love asking people to tell me their *Origin Story*, and have spent a decent amount of time thinking about my own. It makes me happy when people are able to articulate it and are willing to share. I liked that the company valued those things as well.
+- **Independent Coding Problems:** At one onsite, I was given a sheet of questions and an hour to program solutions. Once again, I enjoyed being able to focus all my attention on the problems. It also gave a break from talking, talking, talking. That was nice.
+- **Pass The Tests:** One interview asked me to implement a function to pass a suite of tests. I loved that! As someone who does a lot of TDD, I find it really helpful to have a set of test cases to work through. It helps me concentrate when I can tackle the problem bit by bit.
+- **Pairing:** I did a lot of interviews where I was allowed to google and ask questions. They were described as pairing interviews, but didn't feel like it. They usually involved an interviewer sitting across from me looking at their own laptop... not much different from sitting across from me as I write on a whiteboard. I had one really awesome pairing interview though. We both sat at the same workstation and faced the problem together. As a result, it was much easier to point at things and collaborate. That small change made a big difference.
```

Comment:
definitely my personal opinion: better say "to search the internet" instead of "to google" ;)